### ISSUE TITLE
Suggest `--` when `exec` gets an unrecognised option

### ISSUE THREAD

**Issue by Saviq** [canonical/multipass#531]
When trying to `exec` a command, it's likely to happen that someone forgets to separate options with a `--`. It would be helpful if we suggested its use:

```
$ multipass exec triumphant-gopher lsb_release -a
Unknown option 'a'.
```

Something like:
```
$ multipass exec triumphant-gopher lsb_release -a
Unknown option 'a'.

You need to separate the options to be passed to the command with "--", like this:
multipass exec triumphant-gopher -- lsb_release -a
```

**Comment by dlbeck** [comment]
@Saviq  I am interested in taking on this issue.  I am new to Multipass (and open source in general) and am wondering if you have advice of how I should get start and where to look in the code. I am also open to contribute to a different issue if you think there is one that is more approachable. Thanks!

**Comment by Saviq** [comment]
Hi @dlbeck, welcome to the FOSS train ;)

It should be easy to modify this part of the code to include a note about `--`:

https://github.com/canonical/multipass/blob/4f7ae5cac81ff379f5452a34a20a11e434d2089f/src/client/cli/cmd/exec.cpp#L30L34

**Comment by dlbeck** [comment]
Hi @Saviq I have created a PR for this issue here: #1863 . It fails the `DNSMasqServer.dnsmasq_fails_and_throws` test which seems unrelated to my task. It looks like @katie-knister is having the same issue in PR #1854 so I'm wondering if the issue could be something else. Thanks!

**Comment by ricab** [comment]
Hi @dlbeck , yeah, we keep finding that the timeout that this test depends on is too short for CI. I don't think it's related to your changes. Thanks for your PR!

**Comment by ricab** [comment]
So we discussed a bit, and we think that the message should be a bit simpler than in this issue's description:

```
$ multipass exec triumphant-gopher lsb_release -a
Unknown option 'a'.

Options to the inner command should come after "--", like this:
multipass exec <instance> -- <command> <arguments>
```

**Comment by dlbeck** [comment]
Thanks, that makes sense @ricab . I have made the changes in the PR when you have a chance to review it.

**Comment by Saviq** [comment]
Fixed in #1863
